### ISSUE TITLE
update fails on 23.04 Lunar Lobster

### ISSUE THREAD

**Issue by torusJKL** [wimpysworld/deb-get#819]
### Expected behavior

The local files are updated.
Packages that do not support the current release should be either skipped or updated as is.

### Actual behavior

Update fails because the current version is not supported by all packages.

### Steps to reproduce

1. run `deb-get update`
2. bug reproduces for mergerfs and stops the update process

#### /etc/os-release

```bash
cat /etc/os-release

PRETTY_NAME="Ubuntu 23.04"
NAME="Ubuntu"
VERSION_ID="23.04"
VERSION="23.04 (Lunar Lobster)"
VERSION_CODENAME=lunar
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=lunar
LOGO=ubuntu-logo
```

#### /etc/lsb-release

```bash
cat /etc/lsb-release

DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=23.04
DISTRIB_CODENAME=lunar
DISTRIB_DESCRIPTION="Ubuntu 23.04"

```

#### deb-get version

```bash
deb-get version

0.3.7
```

### deb-get update

```bash
deb-get update

  [+] Updating /etc/deb-get/01-main
/etc/deb-get/01-main.repo             100%[======================================================================>]   2.70K  --.-KB/s    in 0s      
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
100  102k    0  102k    0     0  49912      0 --:--:--  0:00:02 --:--:-- 49912
/home/gal
  [+] Updating cache of supported apps in the background
  [!] ERROR! Missing required information of github package mergerfs:
URL=
VERSION_PUBLISHED=
```

**Comment by philclifford** [comment]
This should already be fixed by #774 
Seemingly related to/duplicate of #756 and #763

**Comment by torusJKL** [comment]
The same error reproduces when installing the latest release from this repository.

`sudo dpkg -i deb-get_0.3.7-1_all.deb`
